datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -337,16 +338,16 @@ fn rountrip_aggregate() -> Result<()> {
             DataType::Float64,
         ))],
         // TODO: See <https://github.com/apache/arrow-datafusion/issues/9028>
```

Comment:
We can remove this now, I originally added it in to provide context on why this case was commented out.

Reply:
Missed it. Thanks for catching it